azure-pipelines.yml

```diff
@@ -132,7 +132,7 @@ stages:
         name: readlineSF
         displayName: 'Download readline secure file'
         inputs:
-          secureFile: 'readline7.0.tar.gz'
+          secureFile: 'readline7.0-ncurses6.4.tar.gz'
```

Comment:
@pramodk This looks good as far as it goes. But is it incomplete in the sense that packaging/python/build_static_readline_osx.bash also needs to be updated to ncurses-6.4.   That is what I did to build the wheel locally.

Reply:
Forgot to add the local change to the commit. Should be good in a minute .